### ISSUE TITLE
[Bug]: Can't resolve 'process/browser' in .../core/dist

### ISSUE THREAD

**Issue by mario-iliev** [blocknative/web3-onboard#1470]
### Current Behavior

Hey guys. After updating version of "@web3-onboard/core": "2.12.0" to "2.13.0" I can't run the project and receive the following:

```
ERROR in ./node_modules/@web3-onboard/core/dist/index-f9d2990f.js 10909:53-60
Module not found: Error: Can't resolve 'process/browser' in '/Users/xxx/xxx/node_modules/@web3-onboard/core/dist'
Did you mean 'browser.js'?
BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@web3-onboard/core/dist/index-f9d2990f.js
Cannot read properties of undefined (reading 'module')
```

I have the 'process/browser' plugin in my webpack config and all fallbacks as per your documentation. Any advice?

### Expected Behavior

Run without errors or add migration documentation.

### What package is effected by this issue?

@web3-onboard/core

### Is this a build or a runtime issue?

Runtime

### Package Version

2.13.0

### Node Version

v16.14.2

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by Adamj1232** [comment]
@mario-iliev can you provide more details on your local env? are you using a front end lib? SSR?

**Comment by mario-iliev** [comment]
@Adamj1232 Yeah sorry I created the issue in a hurry. I'm using React CRA with Webpack 5 but I have control over the Webpack configuration so I've added all polyfills required. No SSR, very basic React project. Sadly it's hard for me to give you reproducable example. What I can give you as a clue is that I receive this error if I update either of those packages: "@web3-onboard/core" OR "@web3-onboard/react". Maybe it's a new dependancy added to both those packages in their latest version.
